src/main/webapp/app/admin/report-management/team-settings/team-config-dialog.controller.js

```diff
@@ -29,27 +30,51 @@
         function clear() {
             $uibModalInstance.close();
         }
+        function checkWebhhokIsExiste(webhook) {
+            for (var index = 0; index < vm.webhookList.length; index++) {
+                if (webhook.id) {
+                    if (webhook.webhookName === vm.webhookList[index].webhookName && webhook.id !== vm.webhookList[index].id) {
+                        return true;
+                    }
+                }
+                else {
+                    if (webhook.webhookName === vm.webhookList[index].webhookName) {
+                        return true;
+                    }
+                }
+            }
+            return false;
+        }
         function addWebhook() {
             var teamConfig = {
                 webhookName: vm.connection.details.webhookName,
                 webhookURL: vm.connection.details.webhookURL
             }
-            ChannelService.createTeamConfig(teamConfig)
-                .then(function (success) {
-                    var info = {
-                        text: "team's config is saved",
-                        title: "Saved"
-                    }
-                    $rootScope.showSuccessToast(info);
-                    vm.clear();
+            if (!checkWebhhokIsExiste(teamConfig)) {
+                ChannelService.createTeamConfig(teamConfig)
+                    .then(function (success) {
+                        var info = {
+                            text: "team's config is saved",
+                            title: "Saved"
+                        }
+                        $rootScope.showSuccessToast(info);
+                        vm.clear();
 
-                }).catch(function (error) {
-                    var info = {
-                        text: error.data.message,
-                        title: "Error"
-                    }
-                    $rootScope.showErrorSingleToast(info);
-                });
+                    }).catch(function (error) {
+                        var info = {
+                            text: error.data.message,
+                            title: "Error"
+                        }
+                        $rootScope.showErrorSingleToast(info);
+                    });
+            }
+            else {
+                var info = {
+                    text: "Webhook already exists with this name",
```

Comment:
this error should come from notification center....can not hard code here...also use code uniqueError in json file